apis/python/src/tiledbsoma/tiledb_object.py

```diff
@@ -23,9 +25,27 @@ class TileDBObject(somacore.SOMAObject, Generic[_HandleType]):
     [lifecycle: experimental]
     """
 
+    @classmethod
+    def open(
+        cls: Type[_Self],
+        uri: str,
+        mode: options.OpenMode = "r",
+        *,
+        context: Optional[SOMATileDBContext] = None,
+        platform_config: Optional[options.PlatformConfig] = None,
+    ) -> _Self:
+        """Opens this specific type of collection."""
```

Comment:
Is this docstring correct?

Reply:
It is (but only when you are calling this on a Collection type).